languages/js/sdk-client/src/client.ts

```diff
@@ -26,10 +27,10 @@ export class BitwardenClient {
     this.client = client;
   }
 
-  async accessTokenLogin(accessToken: string): Promise<void> {
+  async loginAccessToken(accessToken: string): Promise<void> {
     const response = await this.client.run_command(
       Convert.commandToJson({
-        accessTokenLogin: {
+        loginAccessToken: {
           accessToken,
         },
       }),
```

Comment:
Ah, one other note. Changing this will break the example file `example/index.js`. We probably want to update the example with that fix and we should probably include an example using the `password` function as well.